.eslintrc

```diff
@@ -1,11 +1,5 @@
 {
-  "extends": [
-    "react-app",
-    "airbnb",
-    "prettier",
-    "prettier/react",
-    "plugin:jest/all"
-  ],
+  "extends": ["react-app", "prettier", "prettier/react", "plugin:jest/all"],
```

Comment:
@hatched Bundled the removal of Airbnb config here otherwise there was 🚨everywhere.